app/serotracker_sqlalchemy/db_model_config.py

```diff
@@ -27,13 +27,17 @@
     ]
 }
 
-db_tables = ['airtable_source',
+db_tables = ['dashboard_source',
```

Comment:
bit confused -- how was the app working if these were not part of the db model config before?

Reply:
Bit misleading but this object is only being used in the test setup, could probably do without this tbh

Reply:
yes, i think just best to read from the list of tables and not have a static obj